recipes/ncvis-climate/patches/fix-resources-path.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/ncvis.cpp b/src/ncvis.cpp
+--- src/ncvis.cpp
++++ src/ncvis.cpp
+@@ -83,9 +83,9 @@
+ 
+ 	wxString wxstrNcVisResourceDir = wxString(std::getenv("NCVIS_RESOURCE_DIR"));
+ 	if (wxstrNcVisResourceDir.length() == 0) {
+ 		wxFileName wxfn(wxStandardPaths::Get().GetExecutablePath());
+-		wxfn.AppendDir(_T("resources"));
++		wxfn.AppendDir(_T("../share/ncvis/resources"));
```

Comment:
I feel like this patch is not using wx path opérations correctly. It seems to be pretty specific for Linux. Is this really what you want? Is there a better way to do this?